README.md

```diff
@@ -310,32 +317,32 @@ After third-party cookies are removed, `embed.maps.com` could no longer set a co
 Set-Cookie: __Host-locationid=187; SameSite=None; Secure; HttpOnly; Path=/; <b>Partitioned;</b>
 </pre>
 
-When the browser's top-level context is still `shoes.com`, any subsequent request to `embed.maps.com` would include the following header:
+Any subsequent request to `embed.maps.com` whose partition key's domain is `shoes.com` would include the following header:
 
 ```
 Cookie: __Host-locationid=187;
 ```
 
-However, when the browser navigates to a different top-level context then the browser would not send the `Cookie` header above to `embed.maps.com`.
-This gives `embed.maps.com` the capability to store users' favorite `shoes.com` store location, but those preferences would only be accessible to `embed.maps.com` when the top-level context is `shoes.com`.
-This is to ensure that `embed.maps.com` cannot use this cookie to link users' activity across different top-level contexts.
+However, when the browser navigates to a different site, the browser would not send the `Cookie` header above to `embed.maps.com`.
+This gives `embed.maps.com` the capability to store users' favorite `shoes.com` store location, but those preferences would only be accessible to `embed.maps.com` when the top-level site is `shoes.com`.
+This is to ensure that `embed.maps.com` cannot use this cookie to link users' activity across different top-level sites.
 
 #### Third-party customer support widgets
 
 Let us also reconsider the [example](#third-party-customer-service-chat-embed) of `retail.com` which wishes to embed a third-party customer support widget, `support.chat.com`, to help users sign up for an account on their site.
-After third-party cookies are removed, `support.chat.com` can only set a cookie when the top-level context is `retail.com` if that cookie has the `Partitioned` attribute:
+After third-party cookies are removed, `support.chat.com` can only set a cookie when the top-level site is `retail.com` if that cookie has the `Partitioned` attribute:
 
 <pre>
 Set-Cookie: __Host-coversationid=a3e70; SameSite=None; Secure; HttpOnly; Path=/; <b>Partitioned;</b>
 </pre>
 
-When the browser's top-level context is `retail.com`, any request to `support.chat.com` would include the cookie:
+Any request to `support.chat.com` whose partition key's domain is `retail.com` would include the cookie:
```

Comment:
Change domain to site, and make it `https://retail.com`

Reply:
See above comment